mempool/mempool.go

```diff
@@ -1307,9 +1452,28 @@ func (mp *TxPool) maybeAcceptTransaction(tx *dcrutil.Tx, isNew, rateLimit, allow
 		return nil, err
 	}
 
+	if isTicket && mp.hasMempoolInput(tx) {
```

Comment:
Add a comment on top reminding that tickets with inputs in the mempool can't be immediately mined due to consensus rules requiring all ticket inputs to be in stakeholder-approved chains, so we accept them into the "stage pool" instead of the main tx pool.